dashboard/lib/sample_data.rb

```diff
@@ -231,9 +231,9 @@ def self.create_section(options)
   # For the given user id, looks up the storage id, or creates a new one if the user doesn't have one.
   def self.find_or_create_storage_id_for_user_id(user_id)
     environment_check!
-    row = user_storage_ids_table.where(user_id: user_id)
-    return row[:id] if row
-    user_storage_ids_table.insert(user_id: user_id)
+    storage_id = storage_id_for_user_id(user_id)
+    return storage_id if storage_id
+    create_storage_id_for_user(user_id)
   end
```

Comment:
This whole helper method seems like something that should exist in storage_id.rb.  We might even be able to avoid exposing `create_storage_id_for_user` directly?

Reply:
This method looks like it's only used in this `sample_data` file for creating test data

Reply:
The main entry point to creating new storage id records is https://github.com/code-dot-org/code-dot-org/blob/staging/shared/middleware/helpers/storage_id.rb#L94-L96

Reply:
I see.  Maybe we can make it clearer that `create_storage_id_for_user` should only be used for test purposes, either with a comment on the method or by renaming it?